api/src/main/java/org/pmiops/workbench/mandrill/MandrillServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package org.pmiops.workbench.mandrill;
+
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import org.json.JSONObject;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.mandrill.ApiException;
+import org.pmiops.workbench.mandrill.api.MandrillApi;
+import org.pmiops.workbench.mandrill.model.MandrillApiKeyAndMessage;
+import org.pmiops.workbench.mandrill.model.MandrillMessage;
+import org.pmiops.workbench.mandrill.model.MandrillMessageStatuses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import javax.inject.Provider;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Service
+public class MandrillServiceImpl implements MandrillService {
+
+  private static final Logger log = Logger.getLogger(MandrillServiceImpl.class.getName());
+  private final Provider<WorkbenchConfig> configProvider;
+  private final Provider<MandrillApi> mandrillApiProvider;
+
+  @Autowired
+  public MandrillServiceImpl(Provider<WorkbenchConfig> configProvider,
+                             Provider<MandrillApi> mandrillApiProvider) {
+    this.configProvider = configProvider;
+    this.mandrillApiProvider = mandrillApiProvider;
+  }
+
+  @Override
+  public MandrillMessageStatuses sendEmail(MandrillMessage email) {
```

Comment:
I'd like to see this return something more generic than `MandrillMessageStatuses`. It should never return null, and throw an exception if it fails. I don't have a good suggestion for what to return at the moment (looking at this not on a work computer right now). I'll look more into this tomorrow, but I suspect that there are different potential conditions that can be gleaned from the response that might be useful to transform into `ApiException`, or something similar.

Reply:
Looking at this with fresh eyes, I think this can be a void method as mentioned above.